### PR TITLE
Add help text and URLs and make spock questions mo re explicit

### LINKED ISSUE #1: Meaning first Spock Question
I think the first Spock question is not clear:

""Would you prefer to have Python based syntax to execute macros e.g. mv('mot01', 100) instead of mv mot01 100? ""

what is really meant?, if you want to have it instead of the existing one (this is what I would understand from the question) or if you want to have it in addition (this is what I would understand from the suggested answers)?. I would say that either the question has to be formulated in other way or the answers have to be changed.

### PR DESCRIPTION
**Pull request by reszelaz**
Fixes #1 and #3.
I make it as a PR so it will be easier to review if there are typos however I will merge it immediatelly. If you find any typos in these changes feel free to fix them and commit directly.

### CODE REVIEW

File: questionnaire.gs [hunk 1]
```diff
@@ -1078,29 +1236,34 @@ to acquire without the need to define a measurement group")
         "Not now and I doubt I will use them in the future"];
     timerable = form.addMultipleChoiceItem()
         .setTitle(item)
-        .setHelpText("0D experimental channels reads values as fast as possible during the integration time, \
-buffers them and finally calculate the result")
+        .setHelpText("0D experimental channels reads values as fast as possible during the integration time, " +
```
Comment (teresanunez) [inline questionnaire.gs:1239]: ... channels read ...

File: questionnaire.gs [hunk 2]
```diff
@@ -1078,29 +1236,34 @@ to acquire without the need to define a measurement group")
...
+            "buffers them and finally calculate the result.\n " +
```
Comment (teresanunez) [inline questionnaire.gs:1240]: buffer them ...

File: questionnaire.gs [hunk 3]
```diff
@@ -1078,29 +1236,34 @@ to acquire without the need to define a measurement group")
...
+            "More info on 0D experimental channels: www.sardana-controls.org/devel/overview/overview_0D.html")
         .setChoiceValues(choices)
         .setRequired(true);
 
     item = "Would you like to have more calculation options available to apply on the 0D experimental channel's buffer, \
 or maybe even be able to easily add new calculations as plugins?";
+    help = "Currently the only available caluclations are sum and average."
```
Comment (teresanunez) [inline questionnaire.gs:1247]: ... calculations ...

File: questionnaire.gs [hunk 4]
```diff
@@ -1187,7 +1353,7 @@ e.g. motor, psuedo motors, etc.")
         "Not now and I doubt I will use them in the future"];
     form.addMultipleChoiceItem()
         .setTitle(item)
-        .setHelpText("Backlash correcion compensate backlash errors typical to stepper motors")
+        .setHelpText("Backlash correction compensate backlash errors typical to stepper motors.")
```
Comment (teresanunez) [inline questionnaire.gs:1356]: correction compensates ...

File: questionnaire.gs [hunk 5]
```diff
@@ -1197,7 +1363,7 @@ e.g. motor, psuedo motors, etc.")
         "Not now and I doubt I will use them in the future"];
     form.addMultipleChoiceItem()
         .setTitle(item)
-        .setHelpText("Backlash correcion compensate backlash errors typical to stepper motors")
+        .setHelpText("Software limits prevents motion from going beyond the safe range set by the user.")
```
Comment (teresanunez) [inline questionnaire.gs:1366]: ... limits prevent ...

**Comment by reszelaz** [comment]
Hi Teresa,
Thanks for your corrections. I have applied them in https://github.com/sardana-org/sardana-questionnaire/commit/90c42ca682675167d00055b17190f0540f3e4884

**Pull request merged**